wallet-test/src/test/scala/org/bitcoins/wallet/models/SpendingInfoDAOTest.scala

```diff
@@ -41,7 +75,7 @@ class SpendingInfoDAOTest extends BitcoinSWalletTest with WalletDAOFixture {
 
     for {
       created <- WalletTestUtil.insertSegWitUTXO(daos)
-      read <- utxoDAO.read(created.id.get)
+      read <- utxoDAO.read(created.id.get).map(_.map(_.toSpendingInfoDb(null)))
```

Comment:
Should just replace these `null`s with `EmptyScriptPubKey` or something

Reply:
sure